metadata-integration/java/datahub-client/src/main/resources/MetadataChangeProposal.avsc

```diff
@@ -103,9 +103,9 @@
       "doc" : "Descriptor for a change action",
       "symbols" : [ "UPSERT", "CREATE", "UPDATE", "DELETE", "PATCH", "RESTATE" ],
       "symbolDocs" : {
-        "CREATE" : "NOT SUPPORTED YET\ninsert if not exists. otherwise fail",
+        "CREATE" : "insert if not exists. otherwise fail",
```

Comment:
Create implies that it's creating a new resource, but this looks like it's not constraining that. How is this different from UPSERT and what purpose is it serving?

Reply:
Intent is to implement what is described here:
> insert if not exists. otherwise fail

For non-timeseries, I tried to implement this by passing `oldValue -> oldValue.orElse(aspect)` as the `updateLambda`, so I did "insert if not exists, otherwise do nothing" so that should probably be changed, or we should change the description of `CREATE`.

Reply:
Got it, yeah description should be updated. Think it's also worth a comment on the updateLambda since it's non-obvious without digging through a few function calls how it gets utilized. Also it's a bit more nuanced than "do nothing" if its relevant to the intent. It will function as if upserting with the same values, i.e. it will update SystemMetadata (unless it's explicitly specified in the MCP and also the exact same) and for lineage aspects it will update timestamps and go through any other logic in the noOp section if it evolves.

Reply:
Hmm perhaps it's simplest to just error then. Will look into it